Exercise5.js

```diff
@@ -0,0 +1,10 @@
+function calculateFactorial(n){
+    if(n < 0) return 0;
+    let factorial = 1;
+    for (let i = 2; i <= n; i++){
+        factorial *= i;
+    }
+    return factorial;
+}
```

Comment:
The same here, you could have left one empty line before and after the loop

```suggestion
function calculateFactorial(n){
    if(n < 0) return 0;
    let factorial = 1;
    
    for (let i = 2; i <= n; i++){
        factorial *= i;
    }
    
    return factorial;
}
```